components/brave_education/resources/brave_education.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
```

Comment:
so per the naming discussion in general, why have just these three files with `brave_` prefix? There is even `education_page_proxy.ts` in this same directory that doesn't have it. So when do you feel like it should have a prefix vs not have one?

Reply:
updated naming to match upstream